src/olympia/reviewers/tests/test_views.py

```diff
@@ -2656,6 +2665,7 @@ def test_attachment_invalid_upload(self):
         )
         assert response.status_code != 302
         assert AttachmentLog.objects.count() == 0
+        self.assertIn('Unsupported file type', response.content.decode('utf-8'))
```

Comment:
Can you assert the whole error?

I think you'll find it'd be something like `Unsupported file type, please upload an archive file (.txt)` - which is a little wrong as `.txt` files aren't archives.